roofit/roofitcore/src/RooNLLVar.cxx

```diff
@@ -355,7 +355,7 @@ Double_t RooNLLVar::evaluatePartition(std::size_t firstEvent, std::size_t lastEv
             << "\n\t" << resultScalar << std::endl;
       }
 
-      if (alwaysPrint || fabs(carry - carryScalar)/carryScalar > 10.) {
+      if (alwaysPrint || fabs(carry - carryScalar)/carryScalar > 500.) {
```

Comment:
Should we use a fixed value here or something which can be configured ? 
Maybe a comment on this could be useful

Reply:
Same as above:
It only runs when you `#define ROOFIT_CHECK_CACHED_VALUES`. Since it's only for 1 or two roofit developers, hard-coding is probably OK. They can change it anytime without consequences for the outside world.

Reply:
OK, thanks I missed that that variable is used only for debugging